corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -162,6 +163,7 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
         <tr class="active">
           <td><i class="fa" data-bind="css: { 'fa-plus-square': !expanded(), 'fa-minus-square': expanded() }, click: $root.toggleGroup"></i></td>
           <td data-bind="text: name ? name : 'No Group'"></td>
+          <td>{% trans "Mandatory" %}</td>
```

Comment:
I'd think we should use the same wording throughout the UI, whether it's "required" or "mandatory" - though we do use "required" in other parts of HQ, so that'll be more familiar to people.

Reply:
Updated this [here](https://github.com/dimagi/commcare-hq/pull/30890/commits/5796020f460d8a9c814ce83bbd25496a9c72a49a)
For bulk upload, I had used "Required" as the header so this was the only place with "Mandatory".